### ISSUE TITLE
Autosuggest - Migrate from Twitter Typeahead to GitHub AutoComplete

### ISSUE THREAD

**Issue by ewlarson** [geoblacklight/geoblacklight#1292]
### Descriptive summary

Blacklight has removed Twitter's [typeahead.js](https://twitter.github.io/typeahead.js/) library, which we've seen flagged with some accessibility issues in the B1G (missing field labels), and replaced the feature with [GitHub's AutoCompleteElement](https://github.com/github/auto-complete-element) library. We should follow suite in GBL.

### Related work

PR for library swap:
https://github.com/projectblacklight/blacklight/pull/2576

Commit removing typeahead:
https://github.com/projectblacklight/blacklight/commit/9d1c86f957678cebd10ac6d3b16c8843bc8f717c

Commit adding GH AutoComplete:
https://github.com/projectblacklight/blacklight/commit/882379cf36f2248e1c7b3a828b6e19cf1cdebf2a

Base layout implementation: 
https://github.com/projectblacklight/blacklight/blob/13a8122fc6495e52acabc33875b80b51613d8351/app/views/layouts/blacklight/base.html.erb#L22-L24

### Timing / Milestone

Maybe this work should wait until we tackle our official Blacklight v8 release?
